### PR TITLE
Hub updates

### PR DESCRIPTION
**Pull request by jwijay**
**Superficial updates to the following pages:**
* Sign up
* Reset password
* Project Files modals: Gdoc account not linked, unlink Github repository
* Project Sharing
* Account Create
* Account Profile
* Account Members
* Account Teams
* Team Members
* Team Projects

---

**Before: sign up (width)**
Screenshot 2019-05-30 16 25 50

**After: Sign up (width)**
Screenshot 2019-05-30 16 30 39

---

**Before: Reset password (spacing, title)**
Screenshot 2019-05-30 16 25 40

**After: Reset password (spacing, title)**
Screenshot 2019-05-30 16 30 32

---

**Before: Modal (button)**
Screenshot 2019-05-30 16 13 28

**After: Modal (button)**
Screenshot 2019-05-30 16 16 00

---
**Before: Unlink modal (copy wording/grammar)**
Screenshot 2019-05-30 16 27 45

**After: Unlink modal (copy wording/grammar)**
Screenshot 2019-05-30 16 16 23

---

**Before: Sharing (buttons, switch color)**
Screenshot 2019-05-30 16 27 57

**After: Sharing (buttons, switch color)**
Screenshot 2019-05-30 16 17 11

---

**Before: Account Create (typo, spacing)**
Screenshot 2019-05-30 16 28 06

**After: Account Create (typo, spacing)**
Screenshot 2019-05-30 16 19 47

---

**Before: Account Profile (button, card)**
Screenshot 2019-05-30 16 28 16

**After: Account Profile (button, card => box)**
Screenshot 2019-05-30 16 20 08

**After: Account Profile (box hover state fix)**
Screenshot 2019-05-30 16 20 13

---

**Before: Account Members (button, spacing)**
Screenshot 2019-05-30 16 28 23

**After: Account Members (button, spacing)**
Screenshot 2019-05-30 16 20 51

---

**Before: Account Teams (cards, button)**
Screenshot 2019-05-30 16 28 29

**After: Account Teams (cards => boxes, button)**
Screenshot 2019-05-30 16 21 12

---

**Before: Team Members (button, spacing)**
Screenshot 2019-05-30 16 29 15

**After: Team Members (button, spacing)**
Screenshot 2019-05-30 16 21 46

---

**Before: Team Projects (button, spacing)**
Screenshot 2019-05-30 16 29 23

**After: Team Projects (button, spacing)**
Screenshot 2019-05-30 16 22 05

### CODE REVIEW

**Comment by alex-ketch** [comment]
Really great round of cleanup @jwijay, looking much tighter overall! 👍

Some feedback:
- It seems we've been using a left-alignment for buttons for the most part, but the "Account Create" view is an outlier, with the "Create" button aligned to the right.
- For a future PR/design iteration, what do you think about testing a two-column or constrained width layout for places where we have a lot of help text (_Account Create_ and _Account Members_ from the screenshots above), to constrain the line-length a bit, and raise the actionable items in hierarchy/verticality a bit?

File: director/accounts/templates/accounts/account_profile.html [hunk 1]
```diff
@@ -19,35 +19,28 @@
                 </div>
                 <div class="level-right">
                     <p class="level-item">
-                        <a class="button is-success" href="{% url 'project_create' %}?account_id={{ account.pk }}">
+                        <a class="button is-primary is-rounded is-outlined call-to-action" href="{% url 'project_create' %}?account_id={{ account.pk }}">
```
Comment (beneboy) [inline director/accounts/templates/accounts/account_profile.html]: I think `call-to-action` is redundant now? What properties do we need for it – I didn't use it throughout any of my last round of changes.

File: director/accounts/templates/accounts/account_teams.html [hunk 2]
```diff
@@ -15,30 +15,22 @@
         <div class="level-right">
             {% if account_permissions|account_permissions_contain:'administer' %}
                 <p class="level-item">
-                    <a class="button is-success" href="{% url 'account_team_create' account.id %}">
+                    <a class="button is-primary is-rounded is-outlined call-to-action" href="{% url 'account_team_create' account.id %}">
```
Comment (beneboy) [inline director/accounts/templates/accounts/account_teams.html]: CTA again/

**Comment by beneboy** [comment]
> * It seems we've been using a left-alignment for buttons for the most part, but the "Account Create" view is an outlier, with the "Create" button aligned to the right.

Anything that is generated from a Django from is probably doing this. They need to be made consistent.

**Comment by jwijay** [comment]
Thanks for the feedback, guys!

I removed the CTA style from: Account List; Project List; Account Profile (New Project); Account Team (New Team); Sign out modal.

> * It seems we've been using a left-alignment for buttons for the most part, but the "Account Create" view is an outlier, with the "Create" button aligned to the right.

I updated the css so it's left-aligned by default. Pages impacted include:
* Project Settings: Meta-data
* Project FIles: Archive
* Project Files: Github Link
* Project Create
* Account Create
* Settings Change Username

> * For a future PR/design iteration, what do you think about testing a two-column or constrained width layout for places where we have a lot of help text (_Account Create_ and _Account Members_ from the screenshots above), to constrain the line-length a bit, and raise the actionable items in hierarchy/verticality a bit?

👍 That sounds good to me.

**Pull request merged**